images/rootfs.yml.in

```diff
@@ -30,6 +30,9 @@ services:
      image: RSYSLOGD_TAG
      cgroupsPath: /eve/services/rsyslogd
      oomScoreAdj: -999
+   - name: newlogd
```

Comment:
Why are we not removing rsyslogd?

Reply:
@rvs I thought we have a knob to switch back to rsyslog as we soak test newlogd

Reply:
@rvs this is short term, most of the clusters not upgraded to the newlog code yet, also for testing now. will remove later.